core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/ColumnFormatRelation.scala

```diff
@@ -377,10 +378,10 @@ abstract class BaseColumnFormatRelation(
         mode match {
           case SaveMode.Ignore =>
 //            dialect match {
-//              case GemFireXDDialect =>
-//                GemFireXDDialect.initializeTable(table,
+//              case SnappyDataDialect =>
```

Comment:
Why don't we remove it completely?

Reply:
At first glance I thought this should be uncommented but has been removed for quite long.